### PR TITLE
Do not display spinner in CI

### LINKED ISSUE #1968: Hide spinner and progress indicator in CI
We can either detect that we are on CI, or add a flag to hide the spinner and progress indicator if we are in CI

### PR DESCRIPTION
**Pull request by morenol**
This PR adds the change to not display the spinner in the CI. It can also be hidden from the cli with the --hide-spinner flag

Closes #1968

### CODE REVIEW

File: crates/fluvio-cluster/src/render.rs [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+#[derive(Debug)]
+pub enum ProgressRenderer {
+    /// Render the progress using eprintln macro
+    Std,
+    /// Render the progress using Indicatiff
+    Indicatiff(ProgressBar),
+}
+
+impl ProgressRenderer {
+    pub fn println(&self, msg: String) {
```
Comment (sehz) [inline crates/fluvio-cluster/src/render.rs:17]: if this is change to: `pub fn println(&self, msg: &str)` then don't need `into()`

File: crates/fluvio-cluster/src/render.rs [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+        match self {
+            ProgressRenderer::Std => eprintln!("{}", msg),
+            ProgressRenderer::Indicatiff(pb) => pb.println(&msg),
+        }
+    }
+
+    pub fn set_message(&self, msg: String) {
```
Comment (sehz) [inline crates/fluvio-cluster/src/render.rs:24]: same here. `set_message(&self, msg: &str)` would be better

**Review by sehz — CHANGES_REQUESTED** [review]
Looks great.  Couple of minor suggestions.

File: crates/fluvio-cluster/src/render.rs (hunk 2 above)
Comment (morenol) [reply crates/fluvio-cluster/src/render.rs:24]: I was having issues with the fact that set_message in progress bar has as parameter a  `impl Into<Cow<'static, str>>);`.  I will change this to also use this

**Review by sehz — APPROVED** [review]
LGTM

**Pull request merged**